webpack/scenes/ContentViews/Details/Histories/ContentViewHistories.js

```diff
@@ -49,9 +49,9 @@ const ContentViewHistories = ({ cvId }) => {
     const taskType = task ? task.label : taskTypes[action];
 
     if (taskType === taskTypes.removal) {
-      return <>{__('Deleted from ')} <Label isTruncated key="1" color="blue" href={`/lifecycle_environments/${environment?.id}`}>{`${environment?.name ?? __('all environments')}`}</Label>{}</>;
+      return <>{__('Deleted from ')} <Label isTruncated key="1" color="blue" href={`/lifecycle_environments/${environment?.id}`}>{`${environment?.name ?? __('all environments')}`}</Label>{ }</>;
```

Comment:
Was this a prettier change? I've never seen spaces being required between curly braces like this

Reply:
oh, are you using this to insert a literal space via HTML? I feel like CSS margins are a better solution in most cases

Reply:
no this was lint, but looking at this, I think we can delete these {}.

Reply:
a space here does not translate down to the html. You would need to do {' '}.

Reply:
ahhh right :)